WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
For the intended audience, the regulations are surely more useful. For learning a basic concept, I got nowhere in 10 minutes.